events.go

```diff
@@ -74,8 +74,14 @@ type cpuUsage struct {
 	User   uint64   `json:"user"`
 }
 
+type cfs struct {
+	Period uint64 `json:"period,omitempty"`
+	Quota  int64  `json:"quota,omitempty"`
+}
+
 type cpu struct {
 	Usage      cpuUsage   `json:"usage,omitempty"`
+	Scheduling cfs        `json:"cfs,omitempty"`
```

Comment:
Do we want to make that stats output and this the same?  You have `Scheduling` field, `cfs` tag, and `CpuCfs`, `cpu_cfs` in the two different places.

I think we should pick one and stick with it